lib/versions.json: Add Raspberry Pi 4 Model B

leon-anavi opened the pull request:
Add Raspberry Pi 4 Model B which has been released in June 2019.
Tested on Raspberry Pi 4 Model B with 4GB RAM.

Signed-off-by: Leon Anavi <leon@anavi.org>

leon-anavi committed:
lib/versions.json: Add Raspberry Pi 4 Model B

Add Raspberry Pi 4 Model B which has been released in June 2019.
Tested on Raspberry Pi 4 Model B with 4GB RAM.

Signed-off-by: Leon Anavi <leon@anavi.org>

jcane86 commented on lib/versions.json:
```diff
@@ -1,4 +1,11 @@
+   "c03111": {
+      "relDate": "Q2 2019",
+      "model": "4 model B",
+      "PCBRev": "1.0",
```
According to elinux.org, the PCBrev is 1.1.

If you fix that we can merge this in. Thanks for your PR!

jcane86 reviewed (changes requested):
Hi Leon, thanks for the PR!

From what im seeing, the PCBrev should be 1.1, fix that please and I'll merge it.

Cheers!

leon-anavi commented:
Hi @jcane86, thanks, I've updated it. Nice catch. Initially I didn't find the revision on the PCB so I set it to 1.0 now I have updated it to 1.1.

Best regards, Leon

jcane86 commented:
:tada: This PR is included in version 1.1.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/raspi-ver/v/1.1.0)
- [GitHub release](https://github.com/jcane86/raspi-ver/releases/tag/v1.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
